user-service/src/main/java/rs/edu/raf/banka1/mapper/CapitalMapper.java

```diff
@@ -0,0 +1,18 @@
+package rs.edu.raf.banka1.mapper;
+
+import org.springframework.stereotype.Component;
+import rs.edu.raf.banka1.dtos.CapitalDto;
+import rs.edu.raf.banka1.model.Capital;
+@Component
+public class CapitalMapper {
+    public CapitalDto capitalToCapitalDto(Capital capital) {
```

Comment:
Jedno pitanje za ovo(mozda ne razumem specifikaciju). Ovde radimo sa konkretnim kapitalom, rezervisana sredstva su vezana za racun? Zasto ovde imamo polja reserved i total

Reply:
Ako je u pitanju neki listing, onda rezervišemo količinu, to piše u specifikaciji.

Reply:
Meni zvuci na osnovu spec da bi trebalo da kapitali budu vezani za racun, i da za svaki kapital imamo rezervisana sredstva i total. Mozda gresim ali evo odakle mi to:
![image](https://github.com/RAF-SI-2023/Banka-1-Backend/assets/12122441/b4bb7b34-1953-4beb-835c-553ea6687e64)

Reply:
@MarkoNik pročitaj šta piše ispod.

Reply:
@Kovelja009 sredjeni su endpoints za Capital onako po dogovoru od juce